Assets/Scripts/Scenes/InGame/Player/Item/ItemEffectScript/StatueOfRhanEffect.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UniRx;
+
+namespace Scenes.Ingame.Player
+{
+    public class StatueOfRhanEffect : ItemEffect
+    {
+        private bool _isHaveStatue = false;
+        public override void OnPickUp()
+        {
```

Comment:
```suggestion
        private CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
        public override void OnPickUp()
        {
            ItemEffect(_cancellationTokenSource.Token).Forget();
```
もしUniTaskで実装る場合はcancellationTokenを入れないとエラーが出るので注意！